pkg/proxy/userspace/proxier.go

```diff
@@ -131,6 +131,8 @@ func createProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables
 	if proxyPorts == nil {
 		proxyPorts = newPortAllocator(util.PortRange{})
 	}
+	glog.V(2).Info("Tearing down pure-iptables proxy rules.")
```

Comment:
I had to go out to move my sister in to her dorm. Will fix nit asap...
sometime later tonight.
On Aug 12, 2015 5:51 PM, "Mike Danese" <notifications@github.com> wrote:

> In pkg/proxy/userspace/proxier.go
> <https://github.com/kubernetes/kubernetes/pull/12597#discussion_r36918792>
> :
>
> > @@ -131,6 +131,8 @@ func createProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables
> >  	if proxyPorts == nil {
> >  		proxyPorts = newPortAllocator(util.PortRange{})
> >  	}
> > +	glog.V(2).Info("Tearing down pure-iptables proxy rules.")
>
> ok sounds good. I think it's okay to leave it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/12597/files#r36918792>.
>